src/components/PokemonCollection.js

```diff
@@ -1,13 +1,25 @@
-import React from "react";
-import PokemonCard from "./PokemonCard";
-import { Card } from "semantic-ui-react";
+import React from "react"
+import PokemonCard from "./PokemonCard"
+import { Card } from "semantic-ui-react"
+
+function PokemonCollection({ pokemonList, searchedPokemon }) {
```

Comment:
It's preferable to pass just `pokemonList` down and have the filtering handled in the parent component, where the list is already filtered before it's passed down.